### DISCUSSION TITLE
Why does asking a question once result in two requests?

### DISCUSSION THREAD

**Discussion by milomoon** [open-webui/open-webui#2982] [General]
# Bug Report
Why does asking a question once result in two requests?
## Description
asking a question once result in two requests

- **Open WebUI Version:** [0.2-0.3.1]
- **OpenAi API

**Comment by justinh-rahb** [comment]
Title generation is a separate prompt.

**Reply by milomoon** [reply]
I've looked at it carefully, and it's not a request to generate a title

**Reply by justinh-rahb** [reply]
Look more carefully, that IS what it is.
